english/lego-world/lego-world.md

```diff
@@ -3,4 +3,4 @@ But this world is a Lego version of Willy Wonka and the Chocolate Factory.
 You are actually skiing down the slope of a marshmallow mountain made out of Legos!
 You are surrounded by lollipop trees and gingerbread houses.
 [Everything is awesome!](https://youtu.be/etzMjoH0rJw)
-And you just can't help but bust out some dance moves
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.